crates/types/src/req.rs

```diff
@@ -76,10 +71,12 @@ mod test {
         let ser = serde_json::to_string(&req).unwrap();
         let de: SignRequest = serde_json::from_str(&ser).unwrap();
         assert_eq!(req, de);
+
         assert_eq!(
             req.signing_hash(),
-            b256!("eabffc9ed79f68618d3628a804d40199c7888cb5274407ee0ad9ef95c7144d0f")
+            b256!("8c89d2c9e8d725ee335a4f35869a001db64d2f6ce2effe7f09d3ef92f6d251ec")
```

Comment:
Note: These changes modify the signing hash because of addition of `host_block_number` and removal of `sequence` and `confirm_by`.